frontend/src/api/pipelines/__tests__/errorUtils.spec.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable camelcase */
+import { handlePipelineFailures } from '~/api/pipelines/errorUtils';
+import { mockPipelineKF } from '~/__mocks__/mockPipelineKF';
+import { proxyGET } from '~/api/proxyUtils';
+import { NotReadyError } from '~/utilities/useFetchState';
+
+jest.mock('~/api/proxyUtils.ts', () => ({
```

Comment:
I don't understand why you are mocking proxy utils as this isn't used by error utils.
Looks to me like all you need a resolved promise that returns a pipeline or similar.